frontend/src/components/meetings/add/Add.tsx

```diff
@@ -0,0 +1,108 @@
+import { useEffect, useState } from 'react'
+import './Add.css'
+import groupsServices from '../../../services/groups'
+import Group from '../../../models/group/group'
+import { useForm } from 'react-hook-form'
+import Draft from '../../../models/meeting/draft'
+import meetingsServices from '../../../services/meetings'
+import { useNavigate } from 'react-router-dom'
+
+export default function Add(): JSX.Element {
+    const [groups, setGroups] = useState<Group[]>([])
+    
+    useEffect(() => {
+        (async () => {
+            try {
+                const groups = await groupsServices.getAll()
+                setGroups(groups)
+            } catch (error) {
+                alert(error)
+            }
+        })()
+    }, [])
+
+    const navigate = useNavigate()
+    const { register, handleSubmit, formState: { errors }, getValues } = useForm<Draft>()
+
+ 
+    const validateEndTime = (value: Date) => {
+        const startTime = new Date(getValues('meetingStart'))
+        const endTime = new Date(value)
+
+        if (endTime <= startTime) {
+            return 'End time must be after start time'
+        }
+        return true
+    }
+
+    
+    async function submit(draft: Draft) {
+        try {
+            await meetingsServices.add(draft)
+            alert('Meeting added')
+            navigate('/meetings/list')
+        } catch (e) {
+            alert(e)
+        }
+    }
+
+    return (
+        <div className="Add">
+            <form onSubmit={handleSubmit(submit)}>
+                <select
+                    {...register('groupId', {
+                        required: { value: true, message: 'Please select a group' }
+                    })}
+                >
+                    <option value="" disabled selected>
+                        Please select group...
+                    </option>
+                    {groups.map(({ id, name }) => (
+                        <option key={id} value={id}>
+                            {name}
+                        </option>
+                    ))}
+                </select>
+                {errors.groupId && <span className="error">{errors.groupId.message}</span>}
+
+                <input
+                    type="datetime-local"
+                    placeholder="Meeting start"
+                    {...register('meetingStart', {
+                        required: { value: true, message: 'Meeting start is required' }
+                    })}
+                />
+                {errors.meetingStart && <span className="error">{errors.meetingStart.message}</span>}
```

Comment:
you should have used css to attach the error label to the right field. currently it is very misleading
<img width="669" alt="Screenshot 2025-03-20 at 14 33 18" src="https://github.com/user-attachments/assets/a4494b6f-c6fd-481d-a0c8-76bc93dff32e" />